### DISCUSSION TITLE
This phone number is already associated with a Vercel account

### DISCUSSION THREAD

**Discussion by congero-club** [vercel/community#4847] [Account Issue]
### Question

So, in the past, I did have a Vercel account - for which I have no longer access, and used the same phone number I am trying to use again. I get blocked with this message ...

**This phone number is already associated with a Vercel account**

When trying to create a new account.

How can I move forward - can Vercel admin delete my old accounts - they are not used anymore.?

**Comment by amyegan** [comment]
Sorry you ran into this problem. We can't safely discuss account information in public because of security concerns. 

If you haven't already emailed to open a new ticket, please do that to open a more private line of communication. You can find the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support)
